src/audioqueue.ts

```diff
@@ -62,40 +88,48 @@ export function interleave(input, output) {
  * passes. After the setup phase no GC is triggered on either side of the queue.
  */
 export class AudioWriter {
+  private ringbuf: RingBuffer;
+
   /**
    * From a RingBuffer, build an object that can enqueue enqueue audio in a ring
    * buffer.
-   * @constructor
    */
-  constructor(ringbuf) {
+  constructor(ringbuf: RingBuffer) {
```

Comment:
I guess this is a matter of preference but we could change this then remove the declaration for `ringbuf` above. If you like the current style better feel free to resolve.

```suggestion
  constructor(private ringbuf: RingBuffer) {
```

Reply:
This is fine as is, thanks.